module_utils/network_lsr/argument_validator.py

```diff
@@ -1858,6 +1916,19 @@ def _validate_post(self, value, name, result):
     VALIDATE_ONE_MODE_NM = "nm"
     VALIDATE_ONE_MODE_INITSCRIPTS = "initscripts"
 
+    def validate_route_tables(self, connection, idx):
+        for r in connection["ip"]["route"]:
+            if isinstance(r["table"], Util.STRING_TYPE):
+                mapping = IPRouteUtils.get_route_tables_mapping()
```

Comment:
I think this is incorrect. Looking at the implementation of the get_route_tables_mapping() it is a class method. Therefore you cannot call it directly. Normally, you need an IPRouteUtils AFAICS. Thanks!